pkg/endpoints/routes_test.go

```diff
@@ -82,6 +83,21 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
+func TestGetWeb200(t *testing.T) {
+	t.Log("Checking GET route for 200 StatusCode")
```

Comment:
Nit around inconsistent whitespace throughout this method, should be inline with our other Go test files (we're using tabs)